lib/lobstersbot/plugins/salute.rb

```diff
@@ -0,0 +1,17 @@
+module Lobstersbot
+  module Salute
+    @@claws = [ "(V)", "(\\/)", "(\\_/)", "V", "v", "(v)" ]
+    @@faces = [ "_!_!_", ".v.", "_00_" ]
+
+    def salute_user(channel)
+      claws = @@claws.sample 2
+      face = @@faces.sample
+      lobster = claws[0] + face + claws[1]
+      privmsg(lobster, channel)
+    end
+
+    def initialize_salute(memory)
```

Comment:
You can use [`.extended`](https://ruby-doc.org/core-2.6.3/Module.html#method-i-extended) here and in PluggableConnection to do the same thing.

Reply:
Thanks for letting me know. Perhaps then we should just use that and abandon the `initialize` calls.